nautobot/dcim/filters.py

```diff
@@ -254,7 +248,7 @@ class SiteFilterSet(NautobotFilterSet, TenancyFilterSet, StatusModelFilterSetMix
         label="Time zone",
         null_value="",
     )
-    tag = TagFilter()
+    tags = TagFilter()
```

Comment:
Can you please humor me and remove this definition from this filterset and see if it behaves the same?

It's what I referenced yesterday in standup, that `TaggableManager` fields are already automatically mapped to `TagFilter` so I'm hoping it should Just Work™. See:

https://github.com/nautobot/nautobot/blob/4168c370153087ecece310624aba3e3c6086ea1b/nautobot/utilities/filters.py#L570

In other words, as long as "tags" is provided in `Meta.fields` for the filterset, I'm pretty certain that it won't need to be overloaded on the filterset class itself. Thanks!